packages/blockchain/test/reorg.spec.ts

```diff
@@ -170,19 +147,8 @@ tape('reorg tests', (t) => {
       )
 
       await blockchain.putBlocks([block1_low, block2_low])
-      const head_low = await blockchain.getHead()
 
       await blockchain.putBlocks([block1_high, block2_high, block3_high])
-      const head_high = await blockchain.getHead()
-
-      t.ok(
-        head_low.hash().equals(block2_low.hash()),
-        'head on the low chain should equal the low block'
-      )
-      t.ok(
-        head_high.hash().equals(block3_high.hash()),
-        'head on the high chain should equal the high block'
-      )
```

Comment:
Same question here.

(or let me know if this has been settled in some chat or something and I missed)

Reply:
I'm going back to understand why i would have done that.
May have been a blunder

Reply:
👍

Reply:
@holgerd77 

both of the tests in `reorg.spec.ts` test getHead() in a way that can't directly apply to getIteratorHead().

As explained in comments below, getIteratorHead() will actually just return the genesis block if the `iterator has not been run`